test_runner/src/test/kotlin/ftl/shard/ShardTest.kt

```diff
@@ -102,13 +103,20 @@ class ShardTest {
             copy(commonArgs = commonArgs.copy(maxTestShards = 4))
         }
 
```

Comment:
Maybe not exactly the same but something similar
```suggestion
// Although the test passes locally, it occasionally fails on VM's used in GH actions
// Therefore we need a retry approach to verify flakiness
```